app/code/core/Mage/Page/etc/system.xml

```diff
@@ -121,6 +121,9 @@
                         <logo_src translate="label">
                             <label>Logo Image Src</label>
                             <sort_order>10</sort_order>
+                            <frontend_type>image</frontend_type>
```

Comment:
This defines the class `Mage_Adminhtml_Block_System_Config_Form_Field_Image`, which results in broken image links:
<img width="721" height="237" alt="image" src="https://github.com/user-attachments/assets/f3918c81-3a32-4fd4-9084-d27221ee1b20" />